docs/contributing-to-translations.md

```diff
@@ -59,20 +67,50 @@ $ bb lang:missing
 |                           :export-page |                                           Export page |
 |                          :graph-search |                                          Search graph |
 |                       :open-new-window |                                            New window |
-...
 ```
 
-Now, add keys for your language to the translation files, save and rerun the above command. Over time
-you're hoping to have this list drop to zero.
+Now, manually, add keys for your language to the translation files, save and rerun the above command. 
+Over time you're hoping to have this list drop to zero.
 
 Almost all translations are pretty quick. The only exceptions to this are the keys `:tutorial/text` and `:tutorial/dummy-notes`. These reference files that are part of the onboarding tutorial. Most languages don't have this translated. If you are willing to do this, we would be happy to have this translated.
 
-## Fix Untranslated
+### \*nix-like command
 
-There is a lot to translate and sometimes we forget to translate a string. To see what translation keys are still left in English:
+For \*nix translators, since this process is a LITTLE tedious to do it manually, here is a small **command** to process the output of `bb lang:missing` command and leave it almost ready to paste it on the dict files.
```

Comment:
Good idea on this workflow. I'll add a `--copy` to support this as I'd rather not have platform-specific workflows. The option will also split the content to be copied by file so it's easier to do

Reply:
That's cool, I tested and works fine!